src/popup/components/TransactionLabel.vue

```diff
@@ -231,7 +230,7 @@ export default defineComponent({
 
       const translation = transactionListTypes[txType.value!] || transactionTypes[txType.value!];
 
-      if (txType.value?.includes('name')) {
+      if (txType.value && txType.value?.includes('name')) {
```

Comment:
what's the point of doubling the `txType.value` here?

Reply:
`txType.value` will return `false`, if it will fail to find a type, this should be adjust, but in a seperate PR.